release/test-dependencies/expected/test_marketplace_products_2025.Q1.json

```diff
@@ -0,0 +1,19 @@
+{
+   "items":[
+      {
+         "id":1,
+         "src":"",
+         "version":"2024 Q4, 2025 Q1, 2025 Q2, 2025 Q3"
+      },
+      {
+         "id":2,
+         "src":"",
+         "version":"2025 Q4"
+      },
+      {
+         "id":3,
+         "src":"",
+         "version":"2026 Q1, 2026 Q2"
+      }
+   ]
```

Comment:
@davisaints, please keep a single JSON here with the following test cases:

- Only quarterly release
- Only 7.x
- Quarterly release and 7.x
- Empty version